setup.cfg

```diff
@@ -0,0 +1,75 @@
+[metadata]
+name = napari_animation
+version = 0.0.1
+url = https://github.com/napari/napari-animation
+download_url = https://github.com/napari/napari-animation
+license = BSD 3-Clause
+license_file = LICENSE
+description = A plugin to make animations with napari
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = Nicholas Sofroniew
```

Comment:
plenty of people working on this now, could consider using comma separated names?

Reply:
Yep ; I added the Github contributors names to the list, but I don't know what was expected (you're involved but your name isn't there). Are there guidelines for that ?